### ISSUE TITLE
Create a component that will load selected series posts matched by route

### ISSUE THREAD

**Issue by monacodelisa** [monacodelisa/Angular-Headless-Hashnode#221]
# Angular App version

- [x] `angular-v17` - has no UI libraries
- [ ] `angular-v17-modMAT` - modules and Angular Material
- [ ] `angular-v17-AnguMAT` - has Angular Material
- [ ] `angular-v17-PrimeNG` - has PrimeNG

**Additional context**
The idea here is to implement this after #213 where the blog series (topics) are already connected to the actual blog topics.
Then in a designated componene called `series` or `topic` we display the blog posts from that series.  

My idea is to match by route so when the user visits `<their blog url>/series/<series-name>` they land on a page that displays all the post from that series, then in another issue (further step) will be added clicking on the individual post and displaying it's contents just like from the main page. 

So basically `series` or `topic` has the same functionality like the main page 
- you can look et it as a blog post category, category is a more common term for this

If you have other suggestions for implementation feel free to share them.
